### DISCUSSION TITLE
How often strelka runs in the background?

### DISCUSSION THREAD

**Discussion by alkampfergit** [Security-Onion-Solutions/securityonion#2290] [Unsupported Versions] [answered]
Hi,

I've a standalone SO installation that I'm using to gain confidence with this fantastic tool. I got a question, because it happened two times. I've generated a payload with msfvenom, then download from an azure VM and it seems that strelka correctly processed the file, but no alert is generated. 

Then I had to reboot the hypervisor, so I've turned off SO virtual machine then rebooted, and immediately strelka alerts appears in alert console.

It seems that strelka generates the alert when SO starts, but then it has some sort of backround job that runs not so often, because I waited for almost 2 hours and no alert appears.

To recap:
I've downloaded a file at 21:53
No alert until 23:45
Stopped SO
Rebooted Hypervisor
Restart SO

strelka alert are there as soon as the machine was rebooted

![image](https://user-images.githubusercontent.com/358545/101698733-4f8d1800-3a7a-11eb-82b0-b4481a89f145.png)

Maybe alerts are generated every 2 hours? 

Gian Maria.

**Comment by weslambert** [comment, marked as answer]
There is a currently a condition where Strelka will shut itself down after fifteen minutes, Salt should usually pick it up and restart it, but the interval can sometimes be mismatched.  In the next release, you shouldn't see this kind of behavior.

**Reply by alkampfergit** [reply]
THanks.
